functions.js

```diff
@@ -1 +1,121 @@
 // TODO: add your solutions here!
+### 1.  Build your own concatentation
+function combineWords(word1, word2) {
+	return (word1+ " "+word2);
+}
+
+combineWords('Bob', 'Kliger');
+
+### 2.  Repeat a phrase
+function repeatPhrase(phrase, n) {
+    var n
```

Comment:
no need to declare a new variable called n.  You have a parameter ''n". Hardcoded to 5 iterations.